ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -9708,13 +9708,14 @@ keys:
           type: str
           description: 'IPv6_address/Mask
 
-            "ipv6_address_virtual" key will be deprecated in AVD v4.0 in favor of
-            the new "ipv6_address_virtuals"
-
             If both "ipv6_address_virtual" and "ipv6_address_virtuals" are set, all
             addresses will be configured
 
             '
+          deprecation:
+            warning: true
+            new_key: ipv6_address_virtuals
+            remove_in_version: v5.0.0
```

Comment:
do we want to use git tags or collection version for this?

Reply:
it should be collection version. I will fix

Reply:
fixed